core/src/main/java/com/opensymphony/xwork2/ognl/OgnlUtil.java

```diff
@@ -84,27 +77,15 @@ public class OgnlUtil {
     private final OgnlGuard ognlGuard;
 
     private boolean devMode;
-    private boolean enableExpressionCache = true;
+    private boolean enableExpressionCache;
     private boolean enableEvalExpression;
 
-    private Set<String> excludedClasses = emptySet();
```

Comment:
All the configuration is now injected directly into `SecurityMemberAccess` except for devMode configuration